kuma/static/styles/components/home/column-callout.scss

```diff
@@ -27,7 +27,6 @@ $callout-min-height: 170px;
             @include bidi(((right, $grid-spacing, left, auto),));
             height: $callout-icon-width;
             width: $callout-icon-width;
-            background-image: url($path-to-embedded-images + 'promo-sprite.png');
```

Comment:
Is this removal still needed? Should we remove promo-sprite.png as well?

Reply:
Yeah I reckon it makes sense to remove this now. Let me just double check on other uses of the sprite. If there is none, I will remove the image as well and update the PR.